cli/types/tests/cypress-tests.ts

```diff
@@ -98,6 +98,10 @@ cy.wrap({ foo: [1, 2, 3] })
     s
   })
 
+cy.wrap(['foo', 'bar']).its(1)
+
+cy.wrap([()=>{}, ()=>{}]).invoke(1)
```

Comment:
ughh, in this case it should invoke first function in the array?

Reply:
yep. i tried to strongly type this using `ReturnType`, but couldn't get anything to really work, so left it as is